### PR TITLE
[QnA Maker] Deleting old multi-turn sample and fix markdown for promp…

### PR DESCRIPTION
**Pull request by trojenguri**
Deleting Old multi-turn sample
Fixing markdown for prompts
Updated documentation for the Microsoft teams group chat fix

### CODE REVIEW

File: samples/csharp_dotnetcore/49.qnamaker-all-features/README.md [hunk 1]
```diff
@@ -44,6 +44,31 @@ QnA Maker CLI to deploy the model.
...
+    ~~~
+    using Microsoft.Bot.Connector;
+    using System.Text.RegularExpressions;
+    ~~~
+- Modify `OnTurnAsync` function as:
+    ~~~
+    public override async Task OnTurnAsync(ITurnContext turnContext, CancellationToken cancellationToken = default)
+        {
+            // Teams group chat
+            if (turnContext.Activity.ChannelId.Equals(Channels.Msteams))
```
Comment (johnataylor) [inline samples/csharp_dotnetcore/49.qnamaker-all-features/README.md]: you should use the built in Remove Mention functions: https://github.com/microsoft/botbuilder-dotnet/blob/master/libraries/Microsoft.Bot.Schema/ActivityExtensions.cs#L41

Comment (johnataylor) [reply samples/csharp_dotnetcore/49.qnamaker-all-features/README.md]: in fact do we really need this code snippet in the README at all?

Comment (trojenguri) [reply samples/csharp_dotnetcore/49.qnamaker-all-features/README.md]: This is required as with multi-turn dialog, we cache previous prompt questions, and if any one of these are asked by user, we call qnamaker with corresponding id. But with <at> tag, there is no exact match and it cause score discrepancy.    
Please correct me if I am wrong, the code that you shared is removal from Activity class, which is used for output, so how can we modify incoming queries with this function?

File: samples/javascript_nodejs/49.qnamaker-all-features/README.md [hunk 2]
```diff
@@ -58,6 +58,26 @@ QnA knowledge base setup and application configuration steps can be found [here]
...
+- Update `run` function as
+    ~~~
+    async run(context, accessor) {
+        const dialogSet = new DialogSet(accessor);
+        dialogSet.add(this);
+
+        const dialogContext = await dialogSet.createContext(context);
+
+        if (context.activity.channelId === "msteams") {
+            context.activity.text = context.activity.text.replace("/<at>[^<]+<\/at>[ ]*/gi", "");
```
Comment (johnataylor) [inline samples/javascript_nodejs/49.qnamaker-all-features/README.md]: You should use RemoveMentionText extension function.
https://github.com/microsoft/botbuilder-dotnet/blob/master/libraries/Microsoft.Bot.Schema/ActivityExtensions.cs#L64

File: samples/csharp_dotnetcore/49.qnamaker-all-features/README.md [hunk 3]
```diff
@@ -44,6 +44,31 @@ QnA Maker CLI to deploy the model.
...
+            {
+                turnContext.Activity.Text = Regex.Replace(turnContext.Activity.Text, @"<at>[^<]+<\/at>[ ]*", string.Empty);
```
Comment (johnataylor) [inline samples/csharp_dotnetcore/49.qnamaker-all-features/README.md]: Use Activity extension function RemoveMemtionText

https://github.com/microsoft/botbuilder-dotnet/blob/master/libraries/Microsoft.Bot.Schema/ActivityExtensions.cs#L64

Comment (trojenguri) [reply samples/csharp_dotnetcore/49.qnamaker-all-features/README.md]: Done

File: samples/javascript_nodejs/49.qnamaker-all-features/README.md (hunk 2 above)
Comment (trojenguri) [reply samples/javascript_nodejs/49.qnamaker-all-features/README.md]: Done

**Pull request merged**